samplesheets/views.py

```diff
@@ -961,6 +962,205 @@ def handle_alerts_deactivate(cls, irods_request, app_alerts=None):
                 )
             )
 
+    # API Helpers --------------------------------------------------------------
+
+    @classmethod
+    def process_single_accept_request(
```

Comment:
To simplify and keep with conventions, we can simply name this `accept_request()`.